docs/graphql/manual/migrations/reference/metadata-format.rst

```diff
@@ -25,12 +25,239 @@ of multiple files.
+The following files will be generated in the ``metadata/`` directory of your project :
 
-- ``version.yaml``: Contains the metadata version of the server
-- ``tables.yaml``: Contains the metadata related to tables
-- ``remote_schemas.yaml``: Contains the metadata related to :ref:`remote schemas<remote_schemas>`
-- ``functions.yaml``: Contains the metadata related to :ref:`custom functions<custom_sql_functions>`
-- ``allow_list.yaml``: Contains the metadata related to :ref:`allow lists<allow_list>`
-- ``actions.yaml``: Contains the metadata related to :ref:`actions<actions>`
-- ``actions.graphql``: Contains all the action definition and custom type definitions
+version.yaml
+^^^^^^^^^^^^^^
+Contains the metadata version of the server
+
+..  code-block:: yaml
+
+    version: 2
+
+tables.yaml
+^^^^^^^^^^^
+Contains the metadata related to tables
+
+..  code-block::  yaml
+
+    - table:
+        schema: public
+        name: authors
+      array_relationships:
+      - name: books
+        using:
+          foreign_key_constraint_on:
+            column: author_id
+            table:
+              schema: public
+              name: books
+      event_triggers:
+      - name: event_test
+        definition:
+          enable_manual: false
+          insert:
+            columns: '*'
+          delete:
+            columns: '*'
+          update:
+            columns:
+            - id
+            - author_name
+        retry_conf:
+          num_retries: 1
+          interval_sec: 10
+          timeout_sec: 60
+        webhook: <webhook_url>
+    - table:
+        schema: public
+        name: books
+      object_relationships:
+      - name: author
+        using:
+          foreign_key_constraint_on: author_id
+
+
+In above example ``tables.yaml`` contains two tables authors and books. Here webhook_url can be http endpoint that should be triggered or environment variable
+which has http endpoints that should be triggered.
+
+remote_schemas.yaml
+^^^^^^^^^^^^^^^^^^^
+
+Contains the metadata related to :ref:`remote schemas<remote_schemas>`
+
+..  code-block::  yaml
+
+    - name: local
+      definition:
+        url_from_env: REMOTE_SCHEMA
+        timeout_seconds: 40
+
+
+In above example ``remote_schemas.yaml`` contains information about remote schema local where local graphql urls are specified via
+REMOTE_SCHEMA environment variable.
+
+..  code-block::  yaml
+
+    - name: local
+      definition:
+        url: <graphql_url>
+        timeout_seconds: 40
+
+
+The above example is same to the previous except the url is specified directly not via environment variable.
+
+functions.yaml
+^^^^^^^^^^^^^^
+
+Contains the metadata related to :ref:`custom functions<custom_sql_functions>`
+
+..  code-block::  yaml
+
+    - function:
+      schema: public
+      name: search_books
+
+
+In the above example ``functions.yaml`` file consists of search_books custom sql function
```

Comment:
In the above example, the ``functions.yaml`` consists of ``search_books`` custom SQL function